lib/oscar.rb

```diff
@@ -3,6 +3,7 @@ module Oscar
   require 'vagrant-pe_build'
   require 'vagrant-auto_network'
   require 'vagrant-config_builder'
+  require 'vagrant-bolt' unless Vagrant::VERSION < "2.2.0"
```

Comment:
String comparison can be a bit tricksy for version numbers. I'd use the following:

```
require 'rubygems/requirement'
Gem::Requirement.new('>= 2.2.0').satisfied_by?(Gem::Version.new(Vagrant::VERSION))
```

Reply:
Ah good to know. Thanks for the code. I pushed an update with this comparison.